source/Coop.Core/Server/Services/MapEvents/SettlementExitEnterHandler.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Common.Logging;
+using Common.Messaging;
+using Common.Network;
+using Coop.Core.Client.Services.MapEvents.Messages;
+using GameInterface.Services.MapEvents;
+using LiteNetLib;
+using Serilog;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Coop.Core.Server.Services.MapEvents
+{
+    /// <summary>
+    /// Handles changes to parties for settlement entry and exit.
+    /// </summary>
+    public class SettlementExitEnterHandler : IHandler
+    {
+        private readonly IMessageBroker messageBroker;
+        private readonly INetwork network;
+        private readonly ILogger Logger = LogManager.GetLogger<SettlementExitEnterHandler>();
```

Comment:
Also I've been moving any static variables before all instance variables
```suggestion
        private static readonly ILogger Logger = LogManager.GetLogger<SettlementExitEnterHandler>();
```